qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -816,6 +843,21 @@ def _connect_signals(self):
         view.renderProcessTerminated.connect(
             self._on_render_process_terminated)
         view.iconChanged.connect(self.icon_changed)
+        #WORKAROUND for https://bugreports.qt.io/browse/QTBUG-65223
+        if QT_VERSION_STR.startswith('5.10'):
+            page.loadProgress.connect(self._on_load_progress_fake)
+            self.loadProgressFake.connect(self._on_load_progress)
+            self.loadFinishedFake.connect(self._on_history_trigger)
+            self.loadFinishedFake.connect(self._restore_zoom)
+            self.loadFinishedFake.connect(self._on_load_finished)
+            page.loadFinished.connect(self._on_load_finished_fake)
```

Comment:
This makes the name of `_on_load_finished_fake` confusing, as it implies it'd be connected to `loadFinishedFake` (generally, `on_foo` is a reaction to a `foo` signal). I can't think of a really good name though, maybe `_load_finished_workaround` (without `on`) or so?